api/all/src/main/java/io/opentelemetry/api/metrics/DefaultMeter.java

```diff
@@ -174,6 +190,9 @@ public void add(long value, Attributes attributes) {}
 
     @Override
     public void add(long value) {}
+
+    @Override
+    public void remove(Attributes attributes) {}
```

Comment:
Again, this type of change needs to be added to the specification before we can add it to the java API. Like accumulations, the ability to remove attributes seems like a cardinality control that is part of SDK territory rather than API. If you're running into this issue, have you considered using a metric exporter / metric reader that uses delta temporality instead of cumulative? Where delta metrics reset accumulations with each collection, cumulative metrics retain accumulations for the lifecycle of the process and are therefore much more likely to run into issues with cardinality due to retaining accumulations for attributes that haven't been seen in a long time.

The specification needs to take an stance on how cardinality is managed with cumulative metric systems, but hasn't yet done so.